sdk/resourcemanager/README.md

```diff
@@ -33,6 +35,71 @@ If you are an existing user of the older version of Azure management library for
 ```
 [//]: # ({x-version-update-end})
 
+The services available via `azure-resourcemanager` are listed as below:
```

Comment:
Azure class?

Reply:
This section is about the dependency import. It might be better using package name than class name here.